src/utils/context.ts

```diff
@@ -0,0 +1 @@
+import React from 'react';
```

Comment:
This is another controversial pull request hmmmmmm

Reply:
I agree, this is very difficult and requires lots of review.